controllers/machinepools.go

```diff
@@ -74,8 +74,13 @@ func (r *AppWrapperReconciler) scaleMachinePool(ctx context.Context, aw *arbv1.A
 			m[aw.Name] = aw.Name
 			klog.Infof("The instanceRequired array: %v", userRequestedInstanceType)
 
+			machinePoolTaint := cmv1.NewTaint().
+				Key(aw.Name).
```

Comment:
In the unlikely but possible scenarios where the key (aw.name) in one namespace is the same name but on another namespace. What would occur? Should we have the key include the namespace too?